docs/extensions/s3zip/examples/aws-js/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "s3-zip-example",
+    "version": "1.0.0",
+    "main": "main.js",
+    "dependencies": {
```

Comment:
did you forget to add JavaScript code? @vadmeste

Reply:
> did you forget to add JavaScript code? @vadmeste

Yes correct. this is fixed.